source/publish/push_repo.md

```diff
@@ -96,12 +96,94 @@ government services.
 ### GitHub Pages
 
 With some modification, you can also deploy your site with [GitHub
-Pages](https://pages.github.com/), but we do not support this. To do this, you
+Pages](https://pages.github.com/) [external link], but we do not support this. To do this, you
```

Comment:
This is a bit confusing... it's saying one can deploy with GitHub pages but we don't support it. What does it actually mean?

Reply:
This is Ben's content. He spent a while getting github pages working. Other departments have used the TDT formatting and CSS but with a bespoke github pages solution. He wanted to mention it in case people searched for it as a solution

Reply:
I understand why the content is here, I just think the phrasing could be clearer :)